webpack.config.js

```diff
@@ -99,7 +99,9 @@ module.exports = {
     new webpack.DefinePlugin({
         BACKEND_URL: JSON.stringify(process.env.BACKEND_URL || 'http://backend.url.undefined'),
         ENVIRONMENT: JSON.stringify(process.env.ENVIRONMENT || 'dev'),
-        GIT_COMMIT: JSON.stringify(GIT_COMMIT || false)
+        VERIFIER: JSON.stringify(process.env.VERIFIER || false),
+        GIT_COMMIT: JSON.stringify(GIT_COMMIT || false),
+        USH_DISABLE_CHECKS: JSON.stringify(process.env.USH_DISABLE_CHECKS) || ' '
```

Comment:
curious, why do we need the empty space there? (I'm probably missing something obvious sorry) 😬

Reply:
Parsing gets wierd in app.js with an empty string there, I now changed to false instead.